networkit/_NetworKit.pyx

```diff
@@ -7095,18 +7095,29 @@ cdef extern from "<networkit/centrality/Closeness.hpp>":
 
 cdef class Closeness(Centrality):
 	"""
-		Closeness(G, normalized=True, variant=ClosenessVariant::standard/checkConnectdedness=True)
+		Closeness(G, normalized=True, checkConnectdedness=True)
```

Comment:
This notation is highly misleading, as it suggests that the third parameter is a keyword parameter, while it is in fact positional. Invoking `algo = nk.centrality.Closeness(G, normalized=True, checkConnectdedness=True)` is a bug.
Also `normalized` has to be passed, while documentation indicates that it has a default value.

Reply:
Maybe something like?
```
Closeness(G, normalized, bool checkConnectdedness)
Closeness(G, normalized, networkit.centrality.ClosenessVariant variant)
```

Reply:
That's true, thanks for the suggestion.